src/WebJobs.Script/Description/DotNet/PackageManager.cs

```diff
@@ -71,6 +71,8 @@ public Task<PackageRestoreResult> RestorePackagesAsync()
                     Arguments = string.Format(CultureInfo.InvariantCulture, "restore \"{0}\" --packages \"{1}\"", restoreProjectPath, nugetHome)
                 };
 
+                startInfo.Environment.Add("DOTNET_SKIP_FIRST_TIME_EXPERIENCE", "true");
```

Comment:
Since we now have this in multiple places, promoting to a constant would be good.